octokit/users.py

```diff
@@ -0,0 +1,14 @@
+import uritemplate
+
+class Users(object):
+    def get_user(self, inputUser=None, **kwargs):
+        if inputUser:
+            return self.user(user=inputUser)
+        else:
+            return self.current_user
+
+    def get_all_users(self):
+        url = uritemplate.expand(self.user.url, {})
+        if(url[-1] == '/'):
+            url = url[:-1]
+        return self.paginate(url[:-1], {})
```

Comment:
Wait pagination shouldn't take a URL -- it should follow hypermedia. 

@howeik I just noticed this, but resources on their own can't access all of these methods right? (e.g. pagination). Are we still going to do as you suggested and convert all resources into clients?